.github/workflows/native-binary-build-lib.profiler.yml

```diff
@@ -201,10 +201,11 @@ jobs:
 
       #   Build the native binary from C source code
       #
-      #   Use Windows 10 SDK 10.0.20348.0 since build fails with Windows 11 SDK 10.0.22000.0
+      #   (Legacy before october 2025) Use Windows 10 SDK 10.0.20348.0 since build fails with Windows 11 SDK 10.0.22000.0
       #   https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170#vcvarsall-syntax
       #   https://bugs.python.org/issue45220
-      #
+      #    
+      #   (After october 2025) Use default sdk
```

Comment:
i believe we can remove this section otherwise it might cause confusion in future. Probably easier to look through the commit history.